advisories/2024-09-25_cometbft-bls12381-double-hash.md

```diff
@@ -0,0 +1,148 @@
+# BLS12-381 Wrong Signature Generation 
+
```

Comment:
Let's add a short description of the vulnerability as an introduction, including the impact.